Source/JavaScriptCore/inspector/remote/cocoa/RemoteInspectorCocoa.mm

```diff
@@ -179,6 +179,13 @@ static bool canAccessWebInspectorMachPort()
     {
         Locker locker { m_mutex };
 
+        NSLog(@"-------------------------------------");
+        NSLog(@"RemoteInspector::updateAutomaticInspectionCandidate (target)");
+        NSLog(@"m_automaticInspectionEnabled: %s", m_automaticInspectionEnabled ? "YES" : "NO");
+        NSLog(@"TARGET NAME: %s", target->name().utf8().data());
+        NSLog(@"URL: %s", target->url().utf8().data());
```

Comment:
I'm not sure it's okay to log URLs?

Reply:
Won't land this, see https://github.com/WebKit/WebKit/pull/20316#discussion_r1389754551